### DISCUSSION TITLE
In 'Next.js Boilerplate', is this a typo?

### DISCUSSION THREAD

**Discussion by horiyu** [vercel/community#1169] [Help] [answered]
Hi,
I was reading the code for [Next.js Boilerplate](https://vercel.com/templates/next.js/nextjs-boilerplate), the Next.js template, and found what I believe to be a typo. Please let me know if this is indeed a typo.

The current Next.js Boilerplate is as follows:
https://github.com/vercel/vercel/blob/b1c7ca8e91ae8c73eb2514d6d3b657e0dc34c602/examples/nextjs/styles/globals.css#L89

Could it be like this?
`color: rgb(var(--foreground-rgb)); `

Im sorry if there are mistakes🙇‍♂️

**Comment by iteratetograceness** [comment, marked as answer]
@horiyu appreciate this catch! Will be fixed by https://github.com/vercel/next.js/pull/44153.
